packages/components/src/components/icon/library.system.ts

```diff
@@ -40,11 +40,6 @@ export const icons = {
       <path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2Zm0 18a8 8 0 1 1 0-16 8 8 0 0 1 0 16Z"/>
       <path d="M16.373 7.627a1 1 0 0 0-1.414 0l-2.96 2.96-2.96-2.96a1 1 0 0 0-1.413 1.414l-.001-.001 2.96 2.96-2.96 2.96a1 1 0 0 0 1.414 1.414l2.96-2.96 2.96 2.96a1 1 0 0 0 1.414-1.414L13.413 12l2.96-2.96a1 1 0 0 0 0-1.414v.001Z"/>
     </svg>`,
-  confirm: `
-    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-      <path d="M21.528 2.167a1 1 0 0 0-1.384.277l-.002.004-11.333 17-5.127-5.153a1 1 0 0 0-1.413 1.414l-.001-.001 6 6a.997.997 0 0 0 .707.293h.1c.307-.032.57-.199.731-.44l.002-.004 12-18a1 1 0 0 0-.276-1.388l-.004-.002Z"/>
-    </svg>
-  `,
```

Comment:
@mariohamann @karlbaumhauer 
I wanted to remove this confirm icon as it is redundant (status-check already exists, and is preferred by design). 

Would this constitute as a breaking change?

Reply:
thats the question. No one should but everyone could and therewith it IMO is a breaking change by definition. Not by whish... ;-)

Reply:
but I am not 100% sure...

Reply:
I took a look at our documentation that also indicates that system icons should not be used by themselves. 
![image](https://github.com/solid-design-system/solid/assets/63849626/027ac90d-1234-47e9-8cec-b299bc485080)

If we go the breaking change route,by effect, we would be supporting a use case we don't recommend. As this is only a breaking change if you are using the icons where they are not supposed to be. 

I would lean on the side of **not making it a breaking change** and keep an eye out for any issues. Ideally, no one is using them and we can keep treating it as strictly internal.

 Otherwise, we can communicate the change to the respective users that bring this up as an issue, and recommend they replace the icons or we can reevaluate the process then. That's my suggestion. What do you think? @karlbaumhauer @mariohamann

Reply:
thx @Vahid1919. I second what you wrote but still are not 100% sure here... As you described it correctly, it would only be a breaking change if people use our library in a wrong way. 

I asked copilot and got the same answer...🤣
![Screenshot 2024-05-22 at 13 58 48](https://github.com/solid-design-system/solid/assets/48211526/fc521af6-9dc8-4f2d-a326-afdf5b8b58d9)

The only thing which hinders my accepting it as non-breaking:
- we only "recommend" not to use it (instead of not "allowing" it). IMO we, at least, should write that they should not be used and will not be considered by semantic release.
- our changes are not backwards compatible and therewith by definition a breaking change.

Don't get me wrong. Content wise, I totally agree to what you both wrote. I am just struggling with the fact that we haven't been very clear in our docs on that, to just say: "not our issue".

Reply:
I think it's very clear: "As these may change over time, we don't recommend using them directly." I'm clearly on the side on not making it a breaking change.

Otherwise everything could be a breaking change, e. g. private methods which are renamed (but which can still be used), changed markup etc. This would prevent us from making changes at all.

From my opinion it's about how things are documented: CSS Variables, Public Methods etc. are clearly documented to be used – if we change those, it's breaking. Everything else happens under the hood, and that's fine.

Reply:
I am fine with that (and like it) in comparison to private methods.

the only request I would have (for future changes): lets reformulate the docs on that topic. I do not agree, that we are "very clear" as @mariohamann stated. I would love to see something like: "you should never use it as we don't take internal icons into account when it comes to backwards compatibility"...